From 778ec42a36c02da40fd6044ac42fdee0e186c7fc Mon Sep 17 00:00:00 2001
From: Robert Marshall <rmarshall@redhat.com>
Date: Thu, 15 Dec 2016 21:53:31 -0500
Subject: [PATCH 66/69] Clarify --set-default-index in the man page.

This changes the man page to say that the index value provided with
--set-default-index applies to the boot entry list after modifications.

Resolves: rhbz#1285601
---
 grubby.8 | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/grubby.8 b/grubby.8
index 7789b9a..465df32 100644
--- a/grubby.8
+++ b/grubby.8
@@ -144,8 +144,9 @@ boot entry. This may not be invoked with \fB-\-set-default-index\fR.
 
 .TP
 \fB-\-set-default-index\fR=\fIentry-index\fR
-Makes the given entry number the default boot entry. This may not
-be invoked with \fB-\-set-default\fR.
+Makes the given entry number the default boot entry. This may not be invoked
+with \fB-\-set-default\fR.  The given value represents the index in the
+post-modification boot entry list.
 
 .TP
 \fB-\-make-default\fR
-- 
2.9.3

